stripper/ze_border_b2_fix.cfg

```diff
@@ -157,4 +169,29 @@ modify:
 		"OnUser4" "Item_Extinguisher_triggerDisable8-1"
 		"OnUser4" "Item_Extinguisher_triggerKill10-1"
 	}
-}
\ No newline at end of file
+}
+;Fix zombies knifing humans through elevator
+add:
+{
+	"classname"	"trigger_teleport"
+ 	"model" "*26"
+  	"origin" 14063 15699 -1945"
+   	"spawnflags" "4097"
+	"StartDisabled" "1"
+ 	"targetname" "ZM_Elevator_Fix"
+  	"filtername" "Zombies"
+   	"target" "l3_tp_sewer2_d"
+	"UseLandmarkAngles" "1"
+}
+modify:
+{
+	match:
+ 	{
+  		"targetname" "l3_button5"
+	}
+ 	insert:
+  	{
+   		"OnPressed" "ZM_Elevator_Fix,Enable,,20,-1"
+	 	"OnPressed" "ZM_Elevator_Fix,Disable,,30,-1"
+	}
+}
```

Comment:
This would also teleport zombies who made it into the elevator legitimately via a failed defense.